src/Chapter1.hs

```diff
@@ -490,8 +490,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
-
+lastDigit :: Integral a => a -> a
+lastDigit n = (abs n) `mod` 10
```

Comment:
Nice work on using infix form of calling functions! We don't explain this syntax in our course to make it short. But well done :+1: 

Also, you can remove `()`, they are redundant in this case :slightly_smiling_face: 
```suggestion
lastDigit n = abs n `mod` 10
```

Reply:
came in with some advanced knowledge, but not much. I've been reading "Learn You a Haskell" but I'm not very far into it. I've been making a lot of little things to play with the language and then I came across this project which is great. Thanks you for taking the time to review!